src/helpers/config.ts

```diff
@@ -1,25 +1,29 @@
 /* eslint-disable max-lines */
 
-const { yellowBright } = require('chalk')
-const { readJSON, existsSync } = require('fs-extra')
-const { outdent } = require('outdent')
-const { join, dirname, relative } = require('pathe')
-const slash = require('slash')
+import { NetlifyConfig } from '@netlify/build'
+import { yellowBright } from 'chalk'
+import { readJSON } from 'fs-extra'
+import { PrerenderManifest } from 'next/dist/build'
```

Comment:
Can this be `import type`?

Reply:
I can't do `import { PrerenderManifest } from 'next/types'` since they don't have it in their types, is that what you mean?

Reply:
No, I just mean use the type import syntax to make it clear that you're only importing the type:

`import type { PrerenderManifest } from 'next/dist/build'`

Reply:
Ohhh TIL